api/serverpb/server.proto

```diff
@@ -73,9 +80,15 @@ message CheckUpdatesResponse {
   string latest_news_url = 4;
   // Last check time.
   google.protobuf.Timestamp last_check = 5;
+  // True if pmm-update can be used for an update.
+  bool pmm_update_available = 6;
+  // True if pmm-server-upgrade can be used for an update.
+  bool server_upgrade_available = 7;
```

Comment:
Yes. We can end up in a situation where `pmm-server-upgrade` is not running and `pmm-update` is either disabled or not available anymore.
In this situation it would be great if UI shows a message explaining how to deploy `pmm-server-upgrade` for example through a link to docs.

Reply:
ok, got it